modules/KIWILinuxRC.sh

```diff
@@ -750,6 +750,7 @@ function udevStart {
     # start the udev daemon.
     # ----
     local IFS=$IFS_ORIG
+    local fipsEnabled=0
```

Comment:
I would be more in favour of enableFips=0, it would be more in line with e.g enablePlymouth

sorry for nitpicking